polaris-react/src/components/Thumbnail/Thumbnail.scss

```diff
@@ -65,3 +82,17 @@
     fill: currentColor;
   }
 }
+
+.Thumbnail > img {
+  #{$se23} & {
+    position: absolute;
+    top: 50%;
+    right: initial;
+    left: 50%;
+    bottom: initial;
+    width: 100%;
+    height: 100%;
+    transform: translate(-50%, -50%);
+    object-fit: cover;
+  }
```

Comment:
Is it possible to remove these new styles? I think they are zooming in the thumbnails more than before (ignore me if this was intentional!)

|Before|After|After without this block|
|-|-|-|
|<img width="276" alt="Screenshot 2023-06-09 at 4 07 19 PM" src="https://github.com/Shopify/polaris/assets/20652326/6b307caa-b08a-4c19-9e33-c5676aeeba06">|<img width="310" alt="Screenshot 2023-06-09 at 4 07 24 PM" src="https://github.com/Shopify/polaris/assets/20652326/80f36a85-edab-4df8-9770-0910f59c2fd9">|<img width="308" alt="Screenshot 2023-06-09 at 4 12 52 PM" src="https://github.com/Shopify/polaris/assets/20652326/f01b35c0-de78-4c81-b5c3-06260df67eec">|

Reply:
Thanks for letting me know!! just fyi we should also test instances of [ResourceThumbnail](https://web.docs.shopify.io/components/Media%20and%20icons/ResourceThumbnail) in the Admin as it uses the `Thumbnail` component

Reply:
Hey! Yes, this is intentional to match the new [Figma mockups](https://www.figma.com/file/jLLkmo9r28hiqPvf4s90E4/Polaris-Uplift-Components-%5Bgen3%E2%80%93alpha%5D?type=design&node-id=362-1627) so that the image covers the entire background of the thumbnail. It's hard to see the difference with the light image background against the thumbnail background color but here is an example of the before and after with a very bold thumbnail background color so that difference is clearer: 

|Before|After|
|-|-|
| <img width="319" alt="Screenshot 2023-06-09 at 2 36 59 PM" src="https://github.com/Shopify/polaris/assets/21976492/569e672e-728d-46f8-beea-7a4e5ab9c082"> |  <img width="320" alt="Screenshot 2023-06-09 at 2 38 58 PM" src="https://github.com/Shopify/polaris/assets/21976492/fafcc997-b012-43e8-81bb-82f92ae217ce"> |